backend/utils/file_handler.py

```
@@ -28,3 +31,5 @@ def cleanup_temp(path: str) -> None:
         os.remove(real_path)
     except FileNotFoundError:
         pass
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)